system/lib/mimalloc/src/prim/emscripten/prim.c

```diff
@@ -196,7 +201,7 @@ bool _mi_prim_random_buf(void* buf, size_t buf_len) {
 // Thread init/done
 //----------------------------------------------------------------
 
-#ifdef __EMSCRIPTEN_SHARED_MEMORY__
+#if defined(MI_USE_PTHREADS)
```

Comment:
Note: this was changed in https://github.com/microsoft/mimalloc/commit/0b3cd5124999efc673afb26bab3f5a1c8eff4c22#diff-ceeeab7fe916469cce6ab04ba551f6a2411d093f5cb3374cf95f0a1f9bd674aaR203 and https://github.com/microsoft/mimalloc/commit/8f874555d5d42c4e1006bfc78f6cadfb167b1e30#diff-dc7c5d6afe67633e8647bc21fb12adebb183b49dd43e6bd47e1b7a822587fbe0R17.